src/main/java/appeng/integration/modules/helpers/BSCrateStorageAdaptor.java

```diff
@@ -36,8 +36,8 @@
 public class BSCrateStorageAdaptor extends InventoryAdaptor
 {
 
-	final ICrateStorage cs;
-	final ForgeDirection side;
+	private final ICrateStorage cs;
+	private final ForgeDirection side;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "side" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)